src/hevm/hevm-cli/hevm-cli.hs

```diff
@@ -791,6 +800,7 @@ symvmFromCommand cmd = do
       , EVM.vmoptSchedule      = FeeSchedule.istanbul
       , EVM.vmoptChainId       = word chainid 1
       , EVM.vmoptCreate        = create cmd
+      , EVM.vmoptStorageModel  = ConcreteS
```

Comment:
Shoudl this be `model` or `fromMaybe SymbolicS (storageModel cmd)`?

Reply:
yea, this should be `model`. Nice catch